docs/format/index.md

```diff
@@ -43,19 +43,19 @@ Most of the sections described below start with a `len` field. It always specifi
 The symbol table holds a sorted list of deduplicated strings that occurred in label pairs of the stored series. They can be referenced from subsequent sections and significantly reduce the total index size.
 
 The section contains a sequence of the string entries, each prefixed with the string's length in raw bytes. All strings are utf-8 encoded.
-Strings are referenced by pointing to the beginning of their length field. The strings are sorted in lexicographically ascending order.
+Strings are referenced by sequential indexing. The strings are sorted in lexicographically ascending order.
 
 ```
 ┌────────────────────┬─────────────────────┐
 │ len <4b>           │ #symbols <4b>       │
 ├────────────────────┴─────────────────────┤
-│ ┌──────────────────────┬───────────────┐ │
-│ │ len(str_1) <uvarint> │ str_1 <bytes> │ │
-│ ├──────────────────────┴───────────────┤ │
+│ ┌──────────────────┬───────────────────┐ │
+│ │ 0 <uvarint>      │ str_1 <bytes>     │ │
+│ ├──────────────────┴───────────────────┤ │
 │ │                . . .                 │ │
-│ ├──────────────────────┬───────────────┤ │
-│ │ len(str_n) <uvarint> │ str_1 <bytes> │ │
-│ └──────────────────────┴───────────────┘ │
+│ ├──────────────────┬───────────────────┤ │
+│ │ n <uvarint>      │ str_1 <bytes>     │ │
```

Comment:
`len(str)` was  actually the number of bytes we have to subsequently read to know where the string ends. The field was not the offset. 

Looking at the code actually does not serilize the index (correctly) so the update of the docs here seems to be revertible?

Reply:
Ah okay I misread this then. We don't want to include a field for the index in this diagram?